src/org/janelia/mipav/plugins/worm/untwisting/PlotDataUpdateListener.java

```diff
@@ -0,0 +1,30 @@
+package org.janelia.mipav.plugins.worm.untwisting;
+
+import java.util.List;
+
+import WildMagic.LibFoundation.Mathematics.Vector3f;
+
+/**
+ * Interface for handling updates to a plot panel. Classes that need to respond
+ * to plot data changes should implement this interface. An instance of a class
+ * implementing this interface can be registered with components that generate
+ * plot data updates. When plot data changes occur, the {@code updatePlotPanel}
+ * method of each registered listener is called, allowing the implementing class
+ * to update the plot display accordingly.
+ * 
+ * @author diyi chen
+ */
+public interface PlotDataUpdateListener {
+
+	/**
+	 * Invoked to update the plot panel with new data.
+	 * @param points 
+	 * 
+	 * @param values The list of data values to plot. Each value represents a data
+	 *               point on the plot.
+	 * @param title  The title of the plot, describing the data or context of what
+	 *               is being displayed
+	 */
+	void updatePlotPanel(List<Vector3f> points, List<Float> values, String title);
```

Comment:
We should think if we actually need title to be here or not.

Reply:
I think we do because it was string title from either 3-color mode or accurate mode, so if we decide to add 3-color mode, then that will be used to have more descriptive title. Here is where the string title gets the title:

```
		if (m_kVOIInterface.isAccurateMode()) {
			notifyPlotListeners(points, plotAccurateValues, "Selection Chart");
		} else {
			notifyPlotListeners(points, plotValues, "3-Color Chart");
		}
```